extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.bindings.js

```diff
@@ -0,0 +1,87 @@
+angular.module('PaperUI.controllers.configuration.bindings', [ 'ngRoute' ]).config([ '$routeProvider', function($routeProvider) {
+    // Configure the routes for this controller
+    $routeProvider.when('/configuration/bindings', {
```

Comment:
It makes it more modular. It means that the definition of routes sits with the implementation of the code. If someone adds more classes to implement specific additions) then it's all defined in one place.

In my opinion, it's much nicer to keep the code and it's configuration together rather than scattering it across multiple files.